source/broadband/webpa_notification.c

```diff
@@ -56,6 +60,8 @@ static NotifyMsg *notifyMsgQ = NULL;
 void (*notifyCbFn)(NotifyData*) = NULL;
 static WebPaCfg webPaCfg;
 char deviceMAC[32]={'\0'};
+int g_syncRetryThreadStarted = 1;
```

Comment:
Add this as static variable , also initialize this value to 0 first and set to 1 when thread is started .